src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableClient.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.ContextInterfaces
```

Comment:
Change this namespace to `Microsoft.Azure.WebJobs`. We do this for all of our binding related types, as then the customers don't need to add the more complicated namespace just to use the binding.